model/patch/intent.go

```diff
@@ -16,7 +18,7 @@ type Intent interface {
 	Insert() error
 
 	// SetProcessed should be called by an amboy queue after creating a patch from an intent.
-	SetProcessed() error
+	SetProcessed(ctx context.Context) error
```

Comment:
Some of these other funcs will need to be passed in context, but I just did the bare minimum for Update db calls